tests/ConfigTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
Testy na configy píšu taky, ale pokaždý si lámu hlavu nad tím, jestli mám psát test na externí knihovnu, co už je otestovaná. Co myslíš?

Reply:
To jsem měl vlastně spíš napsat u `ConfigDefinitionTest.php`

Reply:
Já to beru tak, že v tom `ConfigDefinitionTest.php` testuju, jestli jsem správně naimplementoval tu definici - tj. jestli konfig, který nemá projít neprojde a který má projít projde.